README.md

```diff
@@ -40,10 +41,16 @@ brief description of what it does.
 * [cubeseer](https://github.com/justinj/cubeseer) - Diagram generator, but contains a cube representation
 * [RaMSDeL](https://github.com/justinj/ramsdel) - Random Move Scrambler Definition Language
 
-#C++
-
+C++
+===
```

Comment:
Ah, I see. I think I'd prefer the escaping, but your call. Either way, does the c++ need this?